krun/platform.py

```diff
@@ -706,6 +706,7 @@ class LinuxPlatform(UnixLikePlatform):
     CSET_CMD = "/usr/bin/cset"
     USER_CSET_DIR = "/cpusets/user"
     RESTRICT_DMESG_FILE = "/proc/sys/kernel/dmesg_restrict"
+    UNKNOWN_SENSOR_CHIP_NAME = "unknown"
```

Comment:
I suggest we make this name a *lot* weirder, because if a real chip has a name "unknown" (which isn't *that* unlikely) our later test will trigger.